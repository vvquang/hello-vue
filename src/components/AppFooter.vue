<template>
  <footer class="footer">
    <div class="footer__content">
      <ul class="footer__content--items">
        <!-- <li><router-link :to="{ name: 'help' }">Help</router-link></li>
        <li><router-link :to="{ name: 'about' }">About</router-link></li> -->
        <li>
          <a :href="`/help`">Help </a>
        </li>
        <li>
          <a :href="`/about`">About</a>
        </li>
      </ul>
      <span class="footer__content--copyright">Â© 2020 1PAC Vietnam Co.,Ltd</span>
    </div>
  </footer>
</template>

<script>
export default {
  name: 'AppFooter'
}
</script>
