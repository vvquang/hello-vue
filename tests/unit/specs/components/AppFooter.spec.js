import { shallowMount } from '@vue/test-utils'
import AppFooter from '@/components/AppFooter.vue'

const wrapper = shallowMount(AppFooter)

describe('AppFooter mount', () => {
  it('should render correctly', () => {
    expect(wrapper.exists()).toBeTruthy()
    expect(wrapper.isVueInstance()).toBeTruthy()
    expect(wrapper).toMatchSnapshot()
  })
})
