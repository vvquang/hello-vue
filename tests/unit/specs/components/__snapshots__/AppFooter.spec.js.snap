// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppFooter mount should render correctly 1`] = `
<footer class="footer">
  <div class="footer__content">
    <ul class="footer__content--items">
      <li><a href="/help">Help </a></li>
      <li><a href="/about">About</a></li>
    </ul> <span class="footer__content--copyright">Â© 2020 1PAC Vietnam Co.,Ltd</span>
  </div>
</footer>
`;
